homeassistant/config_entries.py

```diff
@@ -39,6 +39,9 @@
 # been removed and unloaded.
 SOURCE_UNIGNORE = "unignore"
 
+# This is used to signal that re-authentication is required by the user.
+SOURCE_REAUTH = "reauth"
```

Comment:
This was added in another PR. Looks like this PR here wasn't rebased cleanly.

Reply:
im not always at a computer so i did manually add this constant here, as i the PR order was a bit off with experimenting how reauth should be handled. the actual merge wasnt impactful as it already existed.